.github/workflows/test.yaml

```diff
@@ -206,17 +210,23 @@ jobs:
       run: |
         cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
         SKIP_DIRTY_CHECK=1 GO_BUILD_TAGS=faultinject ./run-checks --unit
+    - name: Merge coverage results
+      if: ${{ steps.cached-results.outputs.already-ran != 'true' && matrix.gochannel != 'latest/stable' }}
+      run: |
+        cd ${{ github.workspace }}/src/github.com/snapcore/snapd || exit 1
+        go install github.com/makholm/covertool
```

Comment:
Go 1.17. complained it needs to be `go install github.com/makholm/covertool@latst` when I installed it locally, but the github job seems happy.